### PR TITLE
[Box, Meta, Mini] Moves clothes from out of equipment lockers and into wardrobe closets

### PR DESCRIPTION
**Pull request by Ikarrus**
##### This PR touches on boxstation, metastation ministations, and several other zlevel .dmms. For my sanity please don't delay too long in merging this.
- Wardrobe closets are not locked, so it should make it a bit easier to steal disguises
- Unclutters equipment lockers
- Heads still have everything in their one locker for convenience, but some of the clothes have been removed if they're already in their department's wardrobe

### CODE REVIEW

**Comment by bustygoku** [comment]
While you're at it, resprite all equipment lockers in the game to have little shelves in them!

**Comment by Dorsisdwarf** [comment]
Fixes #10185

**Comment by Steelpoint** [comment]
This is a good change, makes things a little more tidy.

:+1:

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -31,7 +31,6 @@
 
 /obj/structure/closet/secure_closet/hop/New()
 	..()
-	new /obj/item/clothing/suit/hooded/wintercoat/cargo(src)
```
Comment (RemieRichards) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: MONSTER.

**Pull request merged**